MovieQuiz/Presentation/MovieQuizViewController.swift

```diff
@@ -24,19 +26,19 @@ final class MovieQuizViewController: UIViewController {
         show(quiz: viewModel)
     }
     
-    private func checkAnswer (answer: Bool) -> Bool {
+    private func checkAnswer(answer: Bool) -> Bool {
         if currentQuestion.correctAnswer == answer {
             correctAnswers += 1
         }
         return currentQuestion.correctAnswer == answer
     }
     
-    @IBAction func NoButtonDidTap(_ sender: Any) {
+    @IBAction private func NoButtonDidTap(_ sender: Any) {
         let isCorrect = checkAnswer(answer: false)
         showAnswerResult(isCorrect: isCorrect)
     }
     
-    @IBAction func YesButtonDidTap(_ sender: Any) {
+    @IBAction private func YesButtonDidTap(_ sender: Any) {
```

Comment:
к camelCase привести еще бы, обе функции. Так принято
```suggestion
    @IBAction private func yesButtonDidTap(_ sender: Any) {
```

Reply:
Я поняла, но боюсь, что xcode сломается. Обещаю придерживаться camelCase)